src/Robo/Plugin/Commands/DevelopmentModeCommands.php

```diff
@@ -315,16 +324,18 @@ protected function drushDeployWith(
                 "'drush deploy' command not found. Further work is necessary to support this version of Drush."
             );
         }
+        // After drush deploy, re-import the latest configuration. This includes
+        // the latest configuration_split configuration. Importing this twice
+        // ensures that the latter command enables and disables modules based
+        // upon the most up--to-date configuration. More at:
+        // https://github.com/drush-ops/drush/issues/2449#issuecomment-708655673
+
+        // Currently we only have one local environment type:
+        // LocalDevEnvironmentTypes::DDEV, so no need for other implementations.
         return $this->taskExecStack()
             ->dir("$this->drupalRoot/sites/$siteDir")
-            ->exec("$localEnvironmentType->value drush @$siteDir.$localEnvironmentType->value deploy --yes")
-            // Import the latest configuration again. This includes the latest
-            // configuration_split configuration. Importing this twice ensures that
-            // the latter command enables and disables modules based upon the most up
-            // to date configuration. Additional information and discussion can be
-            // found here:
-            // https://github.com/drush-ops/drush/issues/2449#issuecomment-708655673
-            ->exec("$localEnvironmentType->value drush @$siteDir.$localEnvironmentType->value config:import --yes")
+            ->exec("drush @$siteDir.$localEnvironmentType->value deploy --yes")
+            ->exec("drush @$siteDir.$localEnvironmentType->value config:import --yes")
```

Comment:
I would love help with insights into why we use `$localEnvironmentType->value` here. Shouldn't `drush @$siteDir` be sufficient? 

In other words, if `@$siteDir` and `$siteDir.ddev` are interchangeable, why use `$siteDir.ddev` at all? Simplicity would dictate to just use the site's name.

Reply:
My only guess is to support something like `@siteDir.lando` vs `@siteDir.ddev` vs `@siteDir.docksal` (I made that last one up...I think.)

Reply:
Thank you @mmatsoo. Seems like we could someday probably get rid of all the environment type logic. I am not sure whether to do so in this merge request though, so am going to hold off and not refactor anything other than the exact thing we are trying to address.